kubernetes/samples/scripts/create-weblogic-domain/model-in-image/utils/create-secret.sh

```diff
@@ -22,17 +22,17 @@ function usage() {
   -s <secret-name>    : Name of secret. Required.
 
   -l <key-value-pair> : Secret 'literal' key/value pair, for example
-                        '-l password=abc123'. Can be specified more than once. 
+                        '-l "password=abc 123"'. Can be specified more than once. 
```

Comment:
My bad on this one -  remove space between "abc" and "123".

Reply:
done